src/Drivers/Sparse/NlpSparseEx2Driver.cpp

```diff
@@ -21,7 +21,7 @@ static bool parse_arguments(int argc,
   n = 3;
   inertia_free = false;
   use_cusolver = false;
-  use_ginkgo = true;
```

Comment:
@pelesh : does `use_ginkgo` need to be true by default? It breaks some of existing use cases for sparse ex2.

Reply:
@ashermancinelli and I just came across this bug a moment ago. No, everything should be initialized to `false` and enabled only per user request. If user does not specify the linear solver, HiOp should set its default.